app/models/uphold_connection.rb

```diff
@@ -202,11 +202,21 @@ def valid_card?
     false
   end
 
+  def refresh_token
+    JSON.parse(uphold_access_parameters || '{}').try(:[], 'refresh_token')
```

Comment:
```suggestion
    JSON.parse(uphold_access_parameters || '{}').fetch('refresh_token', nil)
```

Nit: I'm not opposed to using what you're suggested, but I think other parts of the code uses `.fetch` over `.try`.

Reply:
Sure, no problem. Best to be consistent.

Reply:
Done.